pkg/grafana/synthetic-monitoring.go

```diff
@@ -183,24 +185,24 @@ func getProbeList() (*Probes, error) {
 // Check encapsulates a check
 type Check map[string]interface{}
 
-func newCheck(resource grizzly.Resource) Check {
-	return resource.Detail.(Check)
-}
-
-// UID retrieves the UID from a check
-func (c *Check) UID() string {
+// Job retrieves the job name from a check
+func (c *Check) Job() string {
 	job, ok := (*c)["job"]
 	if !ok {
-		return "X"
+		return ""
```

Comment:
Can you add a comment why it would be OK to return an empty string?

Reply:
It absolutely isn't. But these happen in places where functions don't return errors. That does need resolving, no solution yet.